Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:3.6
+# Set up code directory
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+RUN apt-get update && apt-get install -y libssl-dev
+RUN apt-get update && apt-get install -y python3-pip
+RUN apt-get update && apt-get install -y pandoc
+
+#Copy the application py-evm to the /usr/src/app folder
+COPY . /usr/src/app
+
+# Remove existing virtualenv directory
+RUN rm -rf venv_docker_build
```

Comment:
Isn't the point of a dockerfile that it starts from scratch at the beginning of the file, so this folder would never exist?

Reply:
I'm not an expert on Dockerfiles but when this docker file runs to create the image it is based on the actual source code. So whatever you have on disk is taken into account. Hence, you can create an image even from some temporary state that isn't even committed. So it can happen that a previous `venv_docker_build` exists. That btw, is the reason why I'm not using `venv` here to not clash or delete an existing `venv` that may be around.

Reply:
Hm, it still doesn't sound right to me. Maybe you created that virtualenv folder in your host machine outside of docker? Changes in the Dockerfile at line 18 shouldn't propagate back to the host disk, or be still present on a rerun of the Dockerfile.